### DISCUSSION TITLE
SNES adapter jumpers

### DISCUSSION THREAD

**Discussion by VincentEmond** [sanni/cartreader#450] [Need help]
Hello,

I am just wondering what the jumpers on the SNES adapter are? 

There's one at the back labeled CIC and one at the top labeled 5V. Shoul I close them or keep them open?

Thanks!

![20220801_150214](https://user-images.githubusercontent.com/9730015/182226825-ccf30abe-cb18-4b46-a636-c3429742d562.jpg)
![20220801_150139](https://user-images.githubusercontent.com/9730015/182226832-67113a9e-17ed-4c8f-9602-8be6b4675151.jpg)

**Comment by sanni** [comment]
Keep both open.
If you're on HW5(the main PCB with GBA slot) then the switch also doesn't do anything and can be ignored.

**Reply by VincentEmond** [reply]
Perfect Thanks!
